<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="rs-macbook-pro" failures="1" tests="5" name="com.bliss.APITestingFramework.Testcases.CreateCustomerTest" time="12.821" errors="0" timestamp="2022-11-15T23:49:19 GMT" skipped="0">
  <testcase classname="com.bliss.APITestingFramework.Testcases.CreateCustomerTest" name="validateCreateCustomerAPI" time="3.158"/>
  <system-out/>
  <testcase classname="com.bliss.APITestingFramework.Testcases.CreateCustomerTest" name="validateCreateCustomerAPI" time="3.135"/>
  <system-out/>
  <testcase classname="com.bliss.APITestingFramework.Testcases.CreateCustomerTest" name="validateCreateCustomerAPI" time="3.133"/>
  <system-out/>
  <testcase classname="com.bliss.APITestingFramework.Testcases.CreateCustomerTest" name="validateCreateCustomerAPI" time="3.135"/>
  <system-out/>
  <testcase classname="com.bliss.APITestingFramework.Testcases.CreateCustomerTest" name="InvalidateCreateCustomerAPI" time="0.260">
    <failure message="expected [200] but found [401]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [200] but found [401]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:1240)
at org.testng.Assert.assertEquals(Assert.java:1274)
at com.bliss.APITestingFramework.Testcases.CreateCustomerTest.InvalidateCreateCustomerAPI(CreateCustomerTest.java:35)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:80)
at org.testng.internal.invokers.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:17)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
    </failure>
  </testcase> <!-- InvalidateCreateCustomerAPI -->
  <system-out/>
</testsuite> <!-- com.bliss.APITestingFramework.Testcases.CreateCustomerTest -->
